static/views/submit-developer-profile.html

```diff
@@ -904,6 +904,7 @@ <h3>Join the community</h3>
 				</div>
 			</div>
 			<input type="hidden" id="detected-location-id" name="detected-location-id" />
+			<input type="number" name="hourly-rate" id="hourly-rate" placeholder="Hourly Rate" style="width: 100%;" value="{{ .DeveloperProfile.Location }}"/><br />
```

Comment:
```suggestion
```
we probably don't want to ask too many info just at the first time signing up. Hourly rate alongside with other info can be asked later on on the edit section